Library/Formula/gyp.rb

```diff
@@ -0,0 +1,24 @@
+class Gyp < Formula
+  desc "Generate Your Projects"
+  homepage "https://gyp.gsrc.io/"
+  url "https://chromium.googlesource.com/external/gyp",
+    :using => :git,
+    :revision => "7d29c3de1f5eb74330b81b78ea116b67f684b23b"
```

Comment:
We'll need an official upstream tag for this to be admissible. I would be surprised if upstream ever go there, to be honest.

Nothing in the world stopping you adding this to a created tap of your own though, if you wanted to support it there. Some documentation on doing that [here](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md).